### PR TITLE
Version 7 UI fixes

### PR DESCRIPTION
**Pull request by Riippi**
Update some icons.
- Fixes for shadows and form layouts.
- Replace svg based vue component icons with icon font

### CODE REVIEW

File: src/proxy-ui-api/frontend/src/views/Clients/InternalServers/InternalServers.vue [hunk 1]
```diff
@@ -68,9 +68,7 @@
             v-bind:key="certificate.hash"
           >
             <td class="cert-icon">
```
Comment (carohauta) [inline src/proxy-ui-api/frontend/src/views/Clients/InternalServers/InternalServers.vue:70]: Not changed in this PR but this `td` element has different padding compared to the one below (`pl-4 pt-2`)

**Review by carohauta — CHANGES_REQUESTED** [review]

File: src/proxy-ui-api/frontend/src/views/CertificateDetails/CertificateDetails.vue [hunk 2]
```diff
@@ -59,9 +59,9 @@
           @click="showConfirmDelete()"
           data-test="delete-button"
         >
-          <icon-base icon-name="delete" class="icon mr-1"
-            ><icon-declined /></icon-base
-          >{{ $t('action.delete') }}</large-button
+          <i class="icon-Delete key-icon mr-1" />
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/views/CertificateDetails/CertificateDetails.vue]: Fixed. It should have been icon-Declined

File: src/proxy-ui-api/frontend/src/views/Clients/InternalServers/InternalServers.vue (hunk 1 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/views/Clients/InternalServers/InternalServers.vue]: Fixed

**Review by carohauta — APPROVED** [review]
🦄

**Pull request merged**